src/csvcubed/utils/sparql_handler/data_cube_inspector.py

```diff
@@ -30,6 +35,37 @@
 )
 
 
+def _figure_out_end_user_column_type(
+    qube_c: QubeComponentResult, cube_shape: CubeShape
+) -> EndUserColumnType:
+    """This function will decide the columns type for the end user"""
+
+    component_type = ComponentPropertyType(qube_c.property_type)
+
+    if component_type == ComponentPropertyType.Dimension:
+        if qube_c.property == "http://purl.org/linked-data/cube#measureType":
+            return EndUserColumnType.Measures
+
+        return EndUserColumnType.Dimension
+    elif component_type == ComponentPropertyType.Measure:
+        if cube_shape == CubeShape.Pivoted:
+            return EndUserColumnType.Observations
+
+        return EndUserColumnType.Measures
+    elif component_type == ComponentPropertyType.Attribute:
+        if (
+            qube_c.property
+            == "http://purl.org/linked-data/sdmx/2009/attribute#unitMeasure"
```

Comment:
Can you stick this in a centrally defined location and update all references to it to use the constant?